Makefile

```diff
@@ -25,24 +37,37 @@ lint:
 .PHONY: binary
 binary:
 	@echo "WARNING: binary creates a Linux executable. Use cross for macOS or Windows."
+	# TODO(stevvooe): Remove the use of this script. It does nothing. It is
+	# more complicated because the variables need to be converted back from
+	# shell.
 	./scripts/build/binary
 
 .PHONY: cross
 cross:
+	# TODO(stevvooe): Fix the cross build so this isn't a separate target.
+	# Should simply be able to do `make GOOS=windows` to get a cross-compiled
+	# binary.
 	./scripts/build/cross
 
 .PHONY: dynbinary
 dynbinary:
+	# TODO(stevvooe): Change this to target binary but enable cgo, as that is
+	# the only difference.
 	./scripts/build/dynbinary
 
-.PHONY: watch
-watch:
-	./scripts/test/watch
-
 # Check vendor matches vendor.conf
 vendor: vendor.conf
 	vndr 2> /dev/null
-	scripts/validate/check-git-diff vendor
+	if [ "`git status --porcelain -- vendor/ 2>/dev/null`" ]; then \
```

Comment:
> this is the same as if git diff --quiet -- vendor/ 2>/dev/null; then

I tried that myself as well when I was setting this up. It's actually not the same. The important difference is files that are not  checked into git but are created by the previous command. Ex: if `vndr` includes new files, but they aren't in git. `git status` is aware of those files and the check fails, `git diff` does not.

>  It's not quite as concise as the "These files were changed: LIST", but is probably sufficient (and might even be a bit nicer, since it shows you what changed too).

If a lot of files were changed this would be really confusing. Instead of the user seeing a helpful error message and a list of files they would see a huge git diff which would make the error message much less visible.